File: src/IO/dislin-11.0/python/symbols.py
#!/usr/bin/env python
import dislin

ctit = 'Symbols'

dislin.setpag ('da4p')
dislin.metafl ('cons')

dislin.disini ()
dislin.pagera ()
dislin.complx ()
dislin.paghdr ('H. Michels  (', ')', 2, 0)

dislin.height (60)
nl = dislin.nlmess (ctit)
dislin.messag (ctit, (2100 - nl)/2, 200)

dislin.height (50)
dislin.hsymbl (120)

ny = 150

for i in range (0, 22):
  if (i % 4) == 0:
    ny = ny + 400
    nxp = 550
  else:
    nxp = nxp + 350

  nl = dislin.nlnumb (i, -1)
  dislin.number (i, -1, nxp - nl/2, ny + 150)
  dislin.symbol (i, nxp, ny)

dislin.disfin ()

File: src/IO/dislin-11.0/perl/symbols.pl
#!/usr/bin/perl
use Dislin;

$ctit = 'Symbols';

Dislin::setpag ('da4p');
Dislin::metafl ('cons');

Dislin::disini ();
Dislin::pagera ();
Dislin::complx ();
Dislin::paghdr ('H. Michels  (', ')', 2, 0);

Dislin::height (60);
$nl = Dislin::nlmess ($ctit);
Dislin::messag ($ctit, (2100 - $nl)/2, 200);

Dislin::height (50);
Dislin::hsymbl (120);

$ny = 150;
for ($i = 0; $i < 22; $i++) {
    if (($i % 4) == 0) {
        $ny = $ny + 400;
        $nxp = 550;
    }
    else {
	$nxp = $nxp + 350;
    }
    $nl = Dislin::nlnumb ($i, -1);
    Dislin::number ($i, -1, $nxp - $nl/2, $ny + 150);
    Dislin::symbol ($i, $nxp, $ny);
}
Dislin::disfin ();

